tests/libs/sniff_mqtt.py

```
@@ -0,0 +1,219 @@
+        elif message_type == 'external_server':
+            self._print('\nMessage from external server:')
+            self._decode_external_server_message(message.payload)
+        else:
+            self._print(f'\nUnknown message type: {message_type}')
+
+
+    def _decode_module_gateway_message(self, payload: bytes):
+        message = ExternalProtocol_pb2.ExternalClient()
+        message.ParseFromString(payload)
+        if message.HasField('status'):
+            self._handle_status(getattr(message, 'status').deviceStatus)
```

Ruff (0.11.9)

Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)